src/module/io.c

```diff
@@ -9,6 +9,16 @@
 
 #include <stdio.h>
 
+#ifdef _WIN32
+  #include <fcntl.h>
+#endif
```

Comment:
I think this can be a regular unguarded `#include` up with the others. It was probably a mistake to omit it on the other platforms. :)